docs/how-to-guides/customizing-feast/adding-support-for-a-new-online-store.md

```diff
@@ -372,12 +372,28 @@ class RedisOnlineStoreCreator(OnlineStoreCreator):
 ```
 {% endcode %}
 
-3\. You should swap out the `FULL_REPO_CONFIGS` environment variable and run the integration tests against your online store. In the example repo, the file that overwrites `FULL_REPO_CONFIGS` is `feast_custom_online_store/feast_tests.py`, so you would run:
-
-```bash
-export FULL_REPO_CONFIGS_MODULE='feast_custom_online_store.feast_tests'
-make test-python-universal
+3\. Add a Makefile target to the Makefile to run your datastore specific tests by setting the `FULL_REPO_CONFIGS_MODULE` and `PYTEST_PLUGINS` environment variables. You can remove certain tests that are not relevant or still do not work for your datastore using the `-k` option.
+
+{% code title="Makefile" %}
+```Makefile
+test-python-universal-postgres:
```

Comment:
i think this is actually not a good example since the pytest plugin uses the offline store. Maybe sync and copy the cassandra one?

Reply:
fixed